frontend/console/src/features/command-pallete/CommandPalette.tsx

```diff
@@ -25,20 +25,23 @@ export const CommandPalette: React.FC<CommandPaletteProps> = ({ isOpen, onClose
     }
   }, [schemaData])
 
-  const fuse = useMemo(() => {
-    return new Fuse(items, {
-      keys: ['title', { name: 'subtitle', weight: 1 }],
-      minMatchCharLength: 2,
-      fieldNormWeight: 2,
-    })
-  }, [items])
+  const fuseOptions = {
+    keys: ['title', { name: 'subtitle', weight: 1 }],
+    minMatchCharLength: 2,
+    fieldNormWeight: 2,
+  }
```

Comment:
Really nitty optimization but we could pull this out of the function/component now that it's just a constant to avoid it getting reinstantiated every call

Reply:
Yeah we probably have a few places like this 😬